crates/types/src/entities/message.rs

```diff
@@ -37,11 +35,20 @@ pub struct Message {
     pub data: Vec<u8>,
     /// The block height from the parent da layer that originated this message
     pub da_height: DaBlockHeight,
-    /// When the message was spent in the Fuel chain
-    pub fuel_block_spend: Option<BlockHeight>, // TODO: get rid of this
 }
 
-impl Message {
+/// Message send from Da layer to fuel by bridge
+#[cfg_attr(feature = "serde", derive(serde::Serialize, serde::Deserialize))]
+#[derive(Debug, Default, Clone, PartialEq, Eq)]
+pub struct Message {
```

Comment:
I know that it is annoying, but could you duplicate all fields of the `CompressedMessage` here?

We want to make the life of the SDK users as easer as possible.

Reply:
It's flatten on serde though so it is reproduced. The SDK users will consume this as a flattened struct.

Reply:
SDK users will use `fuel-core-types`, they don’t need to create their version of this structure

Reply:
Ok can do